homeassistant/components/verisure/lock.py

```diff
@@ -111,8 +108,11 @@ def changed_method(self) -> str:
 
     @property
     def code_format(self) -> str:
-        """Return the required six digit code."""
-        return "^\\d{%s}$" % self._digits
+        """Return the configured code format."""
```

Comment:
We can leave this as it was so the property doesn't need to fetch it

Reply:
I'm not sure I understand what you mean as `self._digits` is set in `__init__`, unless you are referring to the below comment?

Reply:
I was indeed referring to the below comment